packages/visualizations-react/package.json

```diff
@@ -47,31 +53,60 @@
     },
     "dependencies": {
         "@opendatasoft/visualizations": "^0.9.0",
+        "@testing-library/jest-dom": "^5.11.9",
+        "@testing-library/react": "^11.2.5",
+        "@types/jest": "^29.0.0",
         "use-callback-ref": "^1.2.4"
     },
     "devDependencies": {
-        "@babel/core": "^7.12.10",
-        "@size-limit/preset-small-lib": "^4.9.1",
+        "@babel/core": "^7.19.0",
+        "@babel/preset-env": "^7.19.0",
+        "@babel/preset-react": "^7.18.6",
+        "@babel/preset-typescript": "^7.18.6",
+        "@rollup/plugin-babel": "^5.3.0",
+        "@rollup/plugin-commonjs": "^19.0.0",
+        "@rollup/plugin-json": "^4.1.0",
+        "@rollup/plugin-node-resolve": "^10.0.0",
+        "@rollup/plugin-replace": "^3.0.1",
+        "@rollup/plugin-typescript": "^6.1.0",
+        "@size-limit/preset-small-lib": "^8.1.0",
         "@storybook/addon-essentials": "^6.4.22",
-        "@storybook/addon-info": "^5.3.21",
+        "@storybook/addon-info": "^4.1.18",
         "@storybook/addon-links": "^6.4.22",
         "@storybook/addons": "^6.4.22",
-        "@storybook/react": "^6.4.22",
-        "@testing-library/jest-dom": "^5.11.9",
-        "@testing-library/react": "^11.2.5",
-        "@types/jest": "^26.0.20",
+        "@storybook/react": "^6.1.21",
```

Comment:
Why the downgrade? Feels weird to have a different version that one from storybook